filters/apiusagemonitoring/filter.go

```diff
@@ -37,6 +37,7 @@ const (
 type apiUsageMonitoringFilter struct {
 	Spec  *apiUsageMonitoringSpec
 	Paths []*pathInfo
+	UnknownPath *pathInfo
```

Comment:
@szuecs @ddufourboivin-zalando shouldn't this be private properties?

Reply:
Talked with @maxim-tschumak: for the moment we leave it as it is.

Reply:
You should add json struct tags to the members and have them exported to make the JSON marshal/unmarshal or encode/decode work correctly and reliably